src/api/explorer.js

```diff
@@ -22,46 +22,136 @@ const receiptsQuery = `
     }
 `;
 
-const getApi = () => {
-    return axios.create({
-        baseURL: process.env.REACT_APP_EXPLORER,
-    });
-}
+const getApi = () => axios.create({
+  baseURL: process.env.REACT_APP_EXPLORER,
+});
 
 const palletToAsset = (pallet) => {
-    if (pallet == "ethLLMBridge") {
-        return "LLM";
-    } else if (pallet == "ethLLDBridge") {
-        return "LLD";
-    } else {
-        throw new Error("Unknown pallet");
-    }
-}
+  if (pallet === 'ethLLMBridge') {
+    return 'LLM';
+  } if (pallet === 'ethLLDBridge') {
+    return 'LLD';
+  }
+  throw new Error('Unknown pallet');
+};
 
 export const getSubstrateOutgoingReceipts = async (substrate_address) => {
-    let result = await getApi().post('/graphql', {
-        query: receiptsQuery,
-        variables: {
-            orderBy: ['EVENT_ID_DESC'],
-            filter: {
-                fromId: {
-                    equalTo: substrate_address
+  const result = await getApi().post('/graphql', {
+    query: receiptsQuery,
+    variables: {
+      orderBy: ['EVENT_ID_DESC'],
+      filter: {
+        fromId: {
+          equalTo: substrate_address,
+        },
+      },
+    },
+  });
+
+  const receipts = result.data.data.query.bridgeReceipts.nodes.map((r) => ({
+    asset: palletToAsset(r.event.section),
+    blockHash: r.event.block.hash,
+    amount: r.amount,
+    ethRecipient: r.ethRecipient.id,
+    date: new Date(r.event.block.timestamp).getTime(),
+    receipt_id: r.receiptId,
+  }));
+
+  return receipts.reduce((o, r) => ({ ...o, [r.receipt_id]: r }), {});
+};
+
+export const getDollarsTransfers = async (substrate_address) => {
+  const result = await getApi().post('/graphql', {
+    query: `
+        query Transfers($orderBy: [TransfersOrderBy!], $filter: TransferFilter) {
```

Comment:
Either move this query out to a variable at the top of file or inline the `receiptsQuery` - I don't care either way, but let's be consistent.